python/py-protobuf3/Portfile

```diff
@@ -40,11 +40,12 @@ checksums       sha256  13d3c15ebfad8c28bee203dd4a0f6e600d2a7d2243bac8b5d0e51746
 
 platforms       darwin
 
-python.versions 27
+python.versions 27 34 35 36
```

Comment:
I would skip 34 unless there's a strong usecase (or if this fixes some missing dependency and deprecating all dependent ports should take too much effort). We'll have to start phasing it out, sooner or later (that can happen before the official EOL which is one year from now).

Reply:
Indeed. Policy is now 27, 35, 36, with the default being 36 going forward.

Reply:
Thanks @mojca, yes I've been reading the latest discussions about python on the mailing list. Ok, I'll take advantage of this branch to remove 34 from this port and the new one I've submitted. I'll take that in mind for the future.

Reply:
Thanks @pmetzger

Reply:
Just fyi, 3.4 still seems to be here...

Reply:
@emcrisostomo Just poking again, this one version of `python.versions` still seems off.

Reply:
Thanks @pmetzger, you're right `protobuf3` has not been updated.  I'm fixing it.